backend/package.json

```diff
@@ -1,26 +1,62 @@
 {
-  "dependencies": {
-    "concurrently": "3.4.0",
-    "cross-env": "5.0.0",
-    "express": "4.15.3",
-    "nodemon": "1.11.0"
-  },
-  "devDependencies": {
-    "@types/express": "4.0.35",
-    "@types/node": "7.0.21",
-    "ts-loader": "2.1.0",
-    "tslint": "5.2.0",
-    "tslint-loader": "3.5.3",
-    "typescript": "2.3.2"
-  },
   "main": "index.js",
   "name": "@welovecoding/backend",
+  "version": "0.0.0",
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "concurrently \"tsc -w\" \"cross-env NODE_PATH=./dist nodemon dist/index.js\"",
-    "build": "cross-env NODE_ENV=production && tsc",
-    "start-prod": "cross-env NODE_PATH=./dist && node dist/index.js"
+    "start": "concurrently \"webpack --watch\" \"cross-env NODE_PATH=./dist nodemon dist/main.js\"",
+    "build": "cross-env NODE_ENV=production && webpack",
+    "start-prod": "cross-env NODE_PATH=./dist && node dist/main.js",
+    "database:populate": "cross-env NODE_PATH=./dist && node dist/populate_database.ts.js",
+    "database:reset": "cross-env NODE_PATH=./dist && node dist/reset.ts.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/welovecoding/welovecoding.git"
   },
-  "version": "0.0.0"
+  "keywords": [
```

Comment:
We only need keywords if we plan to publish our application as a library on npm.